### PR TITLE
nova/flavors: Fix Unlist m5.96xlarge & x1.32xlarge

### PR DESCRIPTION
**Pull request by grandchild**
- Use correct vmware extra-specs template
- Move the flavors to _flavors_deleted.tpl

Also, remove the usage of hana_exclusive_contains_legacy_bigvm_flavors config value for hana-exclusive regions, because we don't want to delete flavors anymore if old VMs still refer to them.

### CODE REVIEW

**Comment by grandchild** [comment]
This needs a corresponding secrets change, to remove `hana_exclusive_contains_legacy_bigvm_flavors`. Will be done once this is merged.

File: openstack/sap-seeds/templates/_flavors_deleted.tpl [hunk 1]
```diff
@@ -33,9 +52,30 @@
...
+{{- if .Values.use_hana_exclusive }}
+    {{- tuple . "vmware_hana_exclusive" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
+{{- else }}
+    {{- tuple . "vmware_common" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
+    "host_fraction": "1,0.67,0.34"
+    "resources:CUSTOM_BIGVM": "2"
+    "resources:CUSTOM_MEMORY_RESERVABLE_MB": "1991680"
+{{- end }}
+    "trait:CUSTOM_NUMASIZE_C48_M729": "required"
+    "hw:cpu_cores": "48"  # used in nova-vmware as cores-per-socket (24pCPU = 48vCPU)
```
Comment (fwiesel) [inline openstack/sap-seeds/templates/_flavors_deleted.tpl:80]: I noticed that maybe a bit late, but it says here `hw:cpu_cores=48` vs `vcpus=128` in line 66. Is that intended?

File: openstack/sap-seeds/templates/_flavors_deleted.tpl [hunk 2]
```diff
@@ -24,6 +24,25 @@
...
+{{- if .Values.use_hana_exclusive }}
+    {{- tuple . "vmware_hana_exclusive" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
+{{- else }}
+    {{- tuple . "vmware_common" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
+    "host_fraction": "1/4,3/4,1/2,1"
+    "resources:CUSTOM_BIGVM": "2"
+    "resources:CUSTOM_MEMORY_RESERVABLE_MB": "1468416"
+{{- end }}
+    "trait:CUSTOM_NUMASIZE_C48_M729": "required"
+    "hw:cpu_cores": "48"  # used in nova-vmware as cores-per-socket (24pCPU = 48vCPU)
```
Comment (fwiesel) [inline openstack/sap-seeds/templates/_flavors_deleted.tpl:43]: It says here `"hw:cpu_cores": "48"` vs `vcpus: 90`  in line 29, is that intended?

File: openstack/sap-seeds/templates/_flavors_deleted.tpl (hunk 1 above)
Comment (grandchild) [reply openstack/sap-seeds/templates/_flavors_deleted.tpl:80]: As the comment implies, this number is the cores-per-socket, a.k.a. the vCPU NUMA size for the flavor. So it's always <= the number of vCPUs of the flavor.

File: openstack/sap-seeds/templates/_flavors_deleted.tpl (hunk 2 above)
Comment (grandchild) [reply openstack/sap-seeds/templates/_flavors_deleted.tpl:43]: see above.

File: openstack/sap-seeds/templates/_flavors_deleted.tpl (hunk 1 above)
Comment (grandchild) [reply openstack/sap-seeds/templates/_flavors_deleted.tpl:80]: this is highly confusing in nova -- i thought the comment would clear it up, but since other people have been confused by it too already, even with the comment, i should probably reword it to make it clearer. -> separate PR

Comment (fwiesel) [reply openstack/sap-seeds/templates/_flavors_deleted.tpl:80]: So it is intended. I still surprised that it is a non-integer factor.

**Review by fwiesel — APPROVED** [review]
